static/src/js/schemas/uiSchemas/collections/spatialInformation.jsx

```diff
@@ -0,0 +1,1387 @@
+                  {
+                    'ui:row': [
+                      {
+                        'ui:col': {
+                          md: 12,
+                          children: ['Points']
+                        }
+                      }
+                    ]
+                  },
+                  {
+                    'ui:row': [
+                      {
+                        'ui:col': {
+                          md: 12,
+                          children: ['BoundingRectangles']
+                        }
+                      }
+                    ]
+                  },
+                  {
+                    'ui:row': [
+                      {
+                        'ui:col': {
+                          md: 12,
+                          children: ['GPolygons']
+                        }
+                      }
+                    ]
+                  },
+                  {
+                    'ui:row': [
+                      {
+                        'ui:col': {
+                          md: 12,
+                          children: ['Lines']
+                        }
+                      }
+                    ]
+                  }
+                ]
+              }
+            }
+          ]
+        },
+        Points: {
+          'ui:heading-level': 'h6',
+          items: {
+            'ui:field': 'layout',
+            'ui:layout_grid': {
+              'ui:row': [
+                {
+                  'ui:hide-header': true,
+                  'ui:col': {
+                    md: 12,
+                    children: [
+                      {
+                        'ui:row': [
+                          {
+                            'ui:col': {
+                              md: 6,
+                              children: ['Longitude']
+                            }
+                          }
+                        ]
+                      },
+                      {
+                        'ui:row': [
+                          {
+                            'ui:col': {
+                              md: 12,
+                              children: ['Latitude']
+                            }
+                          }
+                        ]
+                      },
+                      {
+                        'ui:row': [
+                          {
+                            md: 12,
+                            render: (props) => (
+                              <div className="preview-map-link__icon-label">
+                                <PreviewMapLink {...props} type="point" />
+                              </div>
+                            )
+                          }]
+                      }
+                    ]
+                  }
+                }
+              ]
+            }
+          }
+        },
+        BoundingRectangles: {
+          'ui:heading-level': 'h6',
+          items: {
+            'ui:field': 'boundingRectangle'
```

Comment:
This boundingRectangle is not rendering the custom boundingRectangle field
image.png

Reply:
I just did a git pull and am also still not seeing the custom field being rendered. Should look more like this:
![Screenshot 2024-03-11 at 12 14 31 PM](https://github.com/nasa/mmt/assets/135638667/902f7433-7c1a-4912-975b-6415a3e3f06f)